canonical-web-and-design/ubuntu.com
# 204/account users/API integration

## Done

- integrate account users API endpoints
  - update email already exists error
  - update AccountUsers.test.tsx
  - add fetch error handling

_handling of edit and delete will be implemented in another PR_

## QA

1. Go to https://ubuntu-com-10366.demos.haus/advantage/users?test_backend=true and login
2. Add a new user
3. Verify it's been added to the list and persists after refreshing the page
3. Try to add the same user again
4. Verify an error message is displayed ("Cannot add user. User already exists in your organisation.")

## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/204

## Screenshots

![image](https://user-images.githubusercontent.com/7452681/133417223-279054ab-c807-4a68-a365-3489bb597912.png)
